### PR TITLE
kuzco 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by RoseSecurity**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
let's not do the merge commit at all. (no need to create new prs either)

**Comment by RoseSecurity** [comment]
Sorry for the confusion, just cleaned it up

File: Formula/k/kuzco.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class Kuzco < Formula
+  desc "Reviews Terraform and OpenTofu resources and uses AI to suggest improvements"
+  homepage "https://github.com/RoseSecurity/Kuzco"
+  url "https://github.com/RoseSecurity/Kuzco/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "70baccb282e25d74a1d168e3e8e1786431690df9639ca6ef9701f3fdff12a3a4"
+  license "Apache-2.0"
+  head "https://github.com/RoseSecurity/Kuzco.git", branch: "main"
+
+  depends_on "go" => [:build, :test]
```
Comment (chenrui333) [inline Formula/k/kuzco.rb:9]:
```suggestion
  depends_on "go" => :build
  depends_on "opentofu" => :test
```

File: Formula/k/kuzco.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
```
Comment (chenrui333) [inline Formula/k/kuzco.rb:12]:
```suggestion
    ldflags = "-s -w -X github.com/RoseSecurity/kuzco/cmd.Version=#{version}"
    system "go", "build", *std_go_args(ldflags:)
```

Comment (chenrui333) [reply Formula/k/kuzco.rb:12]: need ldflag for the version output

**Review by chenrui333 — APPROVED** [review]

**Review by branchv — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**